project.clj

```diff
@@ -11,6 +11,7 @@
                                [:name "Elango Cheran"]
                                [:email "elango@google.com"]
                                [:timezone "-8"]]]
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [quil "2.2.5"]]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "1.7.170"]
```

Comment:
Once we start using CLJS, should we either include lein-cljsbuild or a build.clj for building the CLJS?

(Thanks for bumping the versions on the deps.  And getting Clojure 1.7.0 to use reader conditionals.)